packages/kmod-6.1-nvidia/nvidia-open-gpu-modules.service.in

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Copy Open GPU drivers to needed modules directory
+Before=load-kernel-modules.service
+After=systemd-tmpfiles-setup.service
+DefaultDependencies=no
```

Comment:
Why `DefaultDependencies=no`? This is usually done to make something happen very early, but because you depend on `systemd-tmpfiles-setup.service`, it won't run until after `local-fs.target` which is essentially what you end up with by using default dependencies.

For most cases of `DefaultDependencies=no` you also need to add `Conflicts=shutdown.target`. It's less important for oneshots but still good to do so that people don't forget for the cases where it does matter.